src/basic/meson.build

```diff
@@ -366,6 +366,34 @@ run_target(
 
 ############################################################
 
+filesystem_includes = ['linux/magic.h',
+                       'linux/gfs2_ondisk.h']
+
+check_filesystems = find_program('check-filesystems.sh')
```

Comment:
these scripts and files were added by a previous commit - the meson integration should be in the same commit